Ports/p7zip/patches/fixes.patch

```diff
@@ -0,0 +1,41 @@
+diff --git a/CPP/7zip/Archive/Zip/ZipItem.cpp b/CPP/7zip/Archive/Zip/ZipItem.cpp
```

Comment:
This patch fails to apply for me:

```
[gunnar@nyx p7zip (p7z)]$ ./package.sh
Installing dependencies of p7zip!
Fetching p7zip!
Downloading URL: https://github.com/jinfeihan57/p7zip/archive/refs/tags/v17.04.tar.gz
p7zip-17.04.tar.gz already exists
Expecting sha256sum: ea029a2e21d2d6ad0a156f6679bd66836204aa78148a4c5e498fe682e77127ef
sha256sum(p7zip-17.04.tar.gz) = 'ea029a2e21d2d6ad0a156f6679bd66836204aa78148a4c5e498fe682e77127ef'
+ tar -xzf p7zip-17.04.tar.gz (nocd)
+ touch .p7zip-17.04.tar.gz_extracted
Patching p7zip!
+ patch -p1
patching file CPP/7zip/Archive/Zip/ZipItem.cpp
patching file CPP/7zip/CMAKE/7za/CMakeLists.txt
Hunk #1 FAILED at 357 (different line endings).
1 out of 1 hunk FAILED -- saving rejects to file CPP/7zip/CMAKE/7za/CMakeLists.txt.rej
patching file CPP/myWindows/config.h
[gunnar@nyx p7zip (p7z)]$
```

Reply:
The one that is failing is the CMakeLists.txt. It seems that the file that is downloaded has CRLF line endings but the patch has LF line endings. I can convert the file to have LF line endings with dos2unix but should I assume people have that installed?